sdk/appconfiguration/app-configuration/README.md

```diff
@@ -1,14 +1,12 @@
-# Azure App Configuration client library for JS
+# App Configuration client library for TypeScript/JavaScript
 
-Azure App Configuration is a managed service that helps developers
-centralize their application configurations, simply and securely.
+[Azure App Configuration][appconfig_concepts] is a managed service that helps developers centralize their application and feature settings simply and securely.
 
-Use the App Configuration client library to:
+Use the client library for App Configuration to:
 
-* create App Configuration settings
-* retrieve the value of a setting for a given key
-* update the value of a setting for a given key
-* delete settings within an App Configuration
+* Create flexible key representations and mappings
+* Tag keys with labels
+* Replay settings from any point in time
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/app-configuration) |
```

Comment:
The API reference docs below this line should point to https://azure.github.io/azure-sdk-for-js/appconfiguration.html

Reply:
Fixed.